packages/graphics/test/Graphics.tests.ts

```diff
@@ -663,7 +663,9 @@ describe('Graphics', function ()
         {
             const graphics = new Graphics();
 
+            graphics.beginFill();
```

Comment:
Changing tests makes me nervous. Could you explain the necessity of adding beginFill/endFill to these Graphics tests?

Reply:
If neither a line style or a fill style is given, no geometry is generated for the shape (the shape is invisible). That's why the tests were failing until I added beginFill/endFill.